sdcm/sct_config.py

```diff
@@ -1130,6 +1130,13 @@ class SCTConfiguration(dict):
              help="Manager agent backup general configuration: checkers, transfers, low_level_retries. "
                   "For example, {'checkers': 100, 'transfers': 2, 'low_level_retries': 20}"),
 
+        dict(name="mgmt_reuse_backup_snapshot", env="SCT_MGMT_REUSE_BACKUP_SNAPSHOT", type=str,
```

Comment:
`mgmt_reuse_backup_snapshot_name`? Otherwise it looks like bool flag.

Reply:
Make sense, fixed